packages/dcos-integration-test/extra/test_auth.py

```diff
@@ -3,16 +3,11 @@
 
 @pytest.fixture(scope='module')
 def auth_cluster(cluster):
-    if not cluster.auth_enabled:
+    if cluster.session.auth is None:
```

Comment:
I agree. I have long been annoyed by the vagueness of the 'cluster' name. It has survived simply for the reason that I didnt want to deal with the fallout of having to cleanup
```
sed -i 's/cluster/cluster_api_client/g' dcos-integration-tests/*
```
but, given what this refactor is about, this seems to be a particularly appropriate time to tackle this